src/test/regress/expected/multi_expanded_modifications.out

```diff
@@ -0,0 +1,30 @@
+CREATE TABLE modifications (
```

Comment:
Because waiting for that comment was easier than searching for the right place myself :) Moved the tests there and rebased.